game/entities/entity_enum.py

```diff
@@ -0,0 +1,23 @@
+from enum import StrEnum
+
+
+class Entities(StrEnum):
+    # tanks
+    LIGHT_TANK = 'light_tank'
+    MEDIUM_TANK = 'medium_tank'
+    HEAVY_TANK = 'heavy_tank'
+    TANK_DESTROYER = 'at_spg'
+    ARTILLERY = 'spg'
+
+    # features
```

Comment:
It seems that tanks, features and etc entities are worth sharing

Reply:
Sharing as in placing the enums in a global namespace?